tox.ini

```diff
@@ -10,102 +10,64 @@ envlist = modification,py{26,33,34,35,36},cover,lint
-# packages installed separately to ensure that downstream deps problems
-# are detected, c.f. #1002
-core_commands =
-    {[base]pip_install} -e acme[dev]
-    nosetests -v acme --processes=-1
-    {[base]pip_install} -e .[dev]
-    nosetests -v certbot --processes=-1 --process-timeout=100
-core_install_args = -e acme[dev] -e .[dev]
-core_paths = acme/acme certbot
-
-plugin_commands =
-    {[base]pip_install} -e certbot-apache
-    nosetests -v certbot_apache --processes=-1 --process-timeout=80
-    {[base]pip_install} -e certbot-nginx
-    nosetests -v certbot_nginx --processes=-1
-plugin_install_args = -e certbot-apache -e certbot-nginx
-plugin_paths = certbot-apache/certbot_apache certbot-nginx/certbot_nginx
-
-dns_plugin_commands =
-    pip install -e certbot-dns-cloudflare
-    nosetests -v certbot_dns_cloudflare --processes=-1
-    pip install -e certbot-dns-digitalocean
-    nosetests -v certbot_dns_digitalocean --processes=-1
-    pip install -e certbot-dns-google
-    nosetests -v certbot_dns_google --processes=-1
-    pip install -e certbot-dns-route53
-    nosetests -v certbot_dns_route53 --processes=-1 --process-timeout=25
-dns_plugin_install_args = -e certbot-dns-cloudflare -e certbot-dns-digitalocean -e certbot-dns-google -e certbot-dns-route53
-dns_plugin_paths = certbot-dns-cloudflare/certbot_dns_cloudflare certbot-dns-digitalocean/certbot_dns_digitalocean certbot-dns-google/certbot_dns_google certbot-dns-route53/certbot_dns_route53
-
-lexicon_dns_plugin_commands =
-    pip install -e certbot-dns-cloudxns
-    nosetests -v certbot_dns_cloudxns --processes=-1
-    pip install -e certbot-dns-dnsimple
-    nosetests -v certbot_dns_dnsimple --processes=-1
-    pip install -e certbot-dns-nsone
-    nosetests -v certbot_dns_nsone --processes=-1
-lexicon_dns_plugin_install_args = -e certbot-dns-cloudxns -e certbot-dns-dnsimple -e certbot-dns-nsone
-lexicon_dns_plugin_paths = certbot-dns-cloudxns/certbot_dns_cloudxns certbot-dns-dnsimple/certbot_dns_dnsimple certbot-dns-nsone/certbot_dns_nsone
-
-compatibility_install_args = -e certbot-compatibility-test
-compatibility_paths = certbot-compatibility-test/certbot_compatibility_test
-
-other_commands =
-    {[base]pip_install} -e letshelp-certbot
-    nosetests -v letshelp_certbot --processes=-1
-    python tests/lock_test.py
-other_install_args = -e letshelp-certbot
-other_paths =  letshelp-certbot/letshelp_certbot tests/lock_test.py
+# pip installs the requested packages in editable mode
+pip_install = {toxinidir}/tools/pip_install_editable.sh
+# pip installs the requested packages in editable mode and runs unit tests in
+# parallel. If CERTBOT_NO_PIN is set not set to 1, packages are installed using
+# certbot-auto's requirements file as constraints.
+install_and_test = {toxinidir}/tools/install_and_test.sh
+py26_packages =
+    acme[dev] .[dev] certbot-apache certbot-nginx certbot-dns-cloudflare certbot-dns-digitalocean certbot-dns-google certbot-dns-route53 letshelp-certbot
```

Comment:
I think formatting this as follows might improve readability of future diffs (and slightly improve readability of the file itself):

```ini
py26_packages = 
    acme[dev] \
    .[dev] \
    certbot-apache \
    certbot-nginx \
    certbot-dns-cloudflare \
    certbot-dns-digitalocean \
    certbot-dns-google \
    certbot-dns-route53 \
    letshelp-certbot
```